kyo-bench/src/main/scala/kyo/bench/TestHttpServer.scala

```diff
@@ -1,64 +1,113 @@
 package kyo.bench
 
-import com.sun.net.httpserver.HttpExchange
-import com.sun.net.httpserver.HttpHandler
-import com.sun.net.httpserver.HttpServer
-import java.io.OutputStream
-import java.net.InetSocketAddress
-import java.util.concurrent.locks.LockSupport
+import io.vertx.core.AbstractVerticle
+import io.vertx.core.DeploymentOptions
+import io.vertx.core.Vertx
+import io.vertx.core.http.HttpServer
+import io.vertx.core.http.HttpServerOptions
+import java.io.BufferedReader
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.io.PrintStream
+import java.io.PrintWriter
+import scala.util.control.NonFatal
 
 object TestHttpServer:
+    val port = 8888
 
     trait StopServer:
         def apply(): Unit
 
-    def log(port: Int, msg: String) =
-        println(s"TestHttpServer(port=$port): $msg")
+    def log(port: Int, msg: String) = println(s"TestHttpServer(port=$port): $msg")
 
-    def start(port: Int): String =
+    def start(concurrency: Int): String =
         val javaBin   = System.getProperty("java.home") + "/bin/java"
         val classpath = System.getProperty("java.class.path")
-        val command   = List(javaBin, "-cp", classpath, "kyo.bench.TestHttpServer", port.toString)
+        val command   = List(javaBin, "-cp", classpath, "kyo.bench.TestHttpServer", concurrency.toString)
         val builder   = new ProcessBuilder(command*)
-        builder.redirectOutput(ProcessBuilder.Redirect.INHERIT)
-        builder.redirectError(ProcessBuilder.Redirect.INHERIT)
-        builder.redirectErrorStream(true)
         try
             log(port, "forking")
             val process = builder.start()
-            val stopServer: StopServer =
-                () =>
-                    log(port, "stopping")
-                    process.destroy()
+            val stopServer: StopServer = () =>
+                log(port, "stopping")
+                process.destroy()
             Runtime.getRuntime().addShutdownHook(new Thread:
-                override def run(): Unit =
-                    stopServer()
+                override def run(): Unit = stopServer()
             )
+            redirect(process.getInputStream, System.out)
+            redirect(process.getErrorStream, System.err)
             s"http://localhost:$port/ping"
         finally
             log(port, "forked")
         end try
     end start
 
+    def redirect(inputStream: InputStream, outputStream: PrintStream): Unit =
```

Comment:
A new mechanism to show that console messages are coming from the forked http server.